phishtray/settings.py

```diff
@@ -174,3 +174,7 @@
 CORS_ALLOW_HEADERS = default_headers + (
     'access-control-allow-origin',
 )
+
+
+# Threshold for percentage number of emails that should have a reveal time of zero
```

Comment:
To be super clear I would add that `The value should be in this range (0.0 - 1.0)`

Reply:
Yes that' a very good point. Thank you!